WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/util/UIUtils.java

```diff
@@ -345,8 +347,15 @@ public static void createOpenReportLink(JComponent parent) {
      * the correct User ID.
      */
     public static void createStartDebugSurveyLink(JComponent parent) {
+        String programmingExperienceParam = "&entry.962486075=";
+        try {
+            programmingExperienceParam += URLEncoder.encode(Preferences.getInstance().getProgrammingExperience(), "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            programmingExperienceParam = "";
+        }
+
         String surveyLink = "https://docs.google.com/forms/d/1ybD1jC-iICXNlmQpyPEFngtmOtodicDr18E1ZbfBtx4/viewform?entry.1872114938="
-                + Preferences.getInstance().getUserId()
+                + Preferences.getInstance().getUserId() + programmingExperienceParam
                 + "&entry.87074017=Java&entry.1002919343=IntelliJ&entry.2010347695&entry.2084367812";
```

Comment:
I think there might be a dynamic way to get the string "Eclipse" or "IntelliJ" (and the brutal way would be to include this in a Plugin-specific class, which would be fine by me) :-)